carma/launch/guidance.launch.py

```diff
@@ -79,6 +79,9 @@ def generate_launch_description():
     inlanecruising_plugin_file_path = os.path.join(
         get_package_share_directory('inlanecruising_plugin'), 'config/parameters.yaml')
 
+     route_following_plugin_file_path = os.path.join(
+        get_package_share_directory('route_following_plugin'), 'config/parameters.yaml')
+
```

Comment:
- this doesn't call the plugin, it just saves variable called `route_following_plugin_file_path` to be used in the actual section where it calls.
- let's call `route_following_plugin` in an existing other file named `plugin.launch.py`. Please launch it using inlanecruising_plugin as an example.
- please don't forget the remaps.